Core/URLExtension.swift

```diff
@@ -33,6 +33,16 @@ extension URL {
     enum Host: String {
         case localhost
     }
+    
+    public var simpleUrl: String {
```

Comment:
(Optional) Not keen on this name but I'm struggling to suggest something better.  I don't like the `Url` part (see `absoluteString`) and it feels like it's doing something quite specific, "simple" being subjective.